packages/three-vrm-core/package.json

```diff
@@ -37,13 +37,16 @@
     "docs": "typedoc --entryPoints ./src/index.ts --out docs",
     "test": "jest",
     "lint": "eslint \"src/**/*.{ts,tsx}\"  && yarn lint-examples &&  prettier \"src/**/*.{ts,tsx}\" --check",
-    "lint-examples": "eslint \"examples/**/*.{ts,tsx,js,html}\" --rule \"padded-blocks: error\"",
-    "lint-fix": "eslint \"src/**/*.{ts,tsx}\" --fix && eslint \"examples/**/*.{ts,tsx,js,html}\" --fix &&  prettier \"src/**/*.{ts,tsx}\" --write"
+    "lint-examples": "eslint \"examples/**/*.{js,html}\" --rule \"padded-blocks: error\"",
+    "lint-fix": "eslint \"src/**/*.{ts,tsx}\" --fix && eslint \"examples/**/*.{js,html}\" --fix &&  prettier \"src/**/*.{ts,tsx}\" --write"
   },
   "lint-staged": {
     "./src/**/*.{ts,tsx}": [
       "eslint --fix",
       "prettier --write"
+    ],
+    "./examples/**/*.{js,html}": [
+      "eslint --fix --rule \"padded-blocks: error\""
```

Comment:
memo: just like ts,tsx files `--fix` option will be applied and potentially change the staged code.